base/applications/atactl/atactl.cpp

```diff
@@ -445,7 +445,7 @@ ata_str_to_mode(
     )
 {
     int mode;
-    int len;
+    size_t len;
```

Comment:
This is code synced with Aleksander Telyatnikov's code (from uniata), so maybe it would be interesting to surround the changes with `#ifdef __REACTOS__ ... #endif`.